assets/css/translation.css

```diff
@@ -81,6 +81,31 @@ html[dir='rtl'] #srcLanguages {
     clear: both;
 }
 
+/* Dictionary Lookup */
+
+#dictionaryLookup {
+    margin-top: 1em;
+}
+
+#dictTranslation {
```

Comment:
Let's not switch between `dict` and `dictionary` arbitrarily. Let's go with the latter?